iOS/FlowWalletKit/Sources/Keys/SecureEnclaveKey.swift

```diff
@@ -86,7 +86,8 @@ public class SecureEnclaveKey: KeyProtocol {
                      signAlgo _: Flow.SignatureAlgorithm = .ECDSA_P256,
                      hashAlgo: Flow.HashAlgorithm) throws -> Data
     {
-        let hashed = try hashAlgo.hash(data: data)
+        let hashed = SHA256.hash(data: data) // try hashAlgo.hash(data: data)
```

Comment:
Need to handle SHA3_256 hash method

Reply:
There will be no SHA3。
on this key is SecureEnclave.P256.Signing.PrivateKey
If the parameter is of type Data, it will be SHA256 in this method,so this causes it to be hash'd twice. 

`public func signature<D>(for data: D) throws -> P256.Signing.ECDSASignature where D : DataProtocol`